pkgs/by-name/on/onedrivegui/package.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, writeText
+, copyDesktopItems
+, makeDesktopItem
+, makeWrapper
+, onedrive
+}:
+
+let
+  version = "1.0.3";
+
+  setupPy = writeText "setup.py" ''
+    from setuptools import setup
+    setup(
+      name='onedrivegui',
+      version='${version}',
+      scripts=[
+        'src/OneDriveGUI.py',
+      ],
+    )
+  '';
+
+in
+python3Packages.buildPythonApplication rec {
+  pname = "onedrivegui";
+  inherit version;
+  disabled = python3Packages.pythonOlder "3.0";
```

Comment:
Do we know which exact version (3.x) is needed?

Reply:
> Do we know which exact version (3.x) is needed?

No, there is no mention of minimum python version that I could find. I don't know what a sensible default would be so I just went with 3.0. Happy to set to something else.